neo/sys/osx/macosx_misc.mm

```diff
@@ -74,7 +74,7 @@ void Sys_DoStartProcess( const char *exeName, bool dofork = true ) {
 */
 const char* OSX_GetLocalizedString( const char* key )
 {
-	NSString *string = [ [ NSBundle mainBundle ] localizedStringForKey:[ NSString stringWithCString: key ]
+	NSString *string = [ [ NSBundle mainBundle ] localizedStringForKey:[ NSString stringWithCString: key encoding:NSUTF8StringEncoding ]
 													 value:@"No translation" table:nil];
 	return [string UTF8String];
```

Comment:
Hmm I wonder if it's even correct to use UTF-8 here (I know this was already the case before your change).
Because in the end it's used in `idKeyInput::KeyNumToString()` which creates key names that are supposed to be used in the menu and printed in the Doom3 console etc - and those don't support UTF-8, but only ISO-8859-1 aka latin-1, so maybe this function should use `NSISOLatin1StringEncoding` somehow?
(Is this function correct anyway? Where/how is the returned string allocated? Who frees it? Because this is called from C++ code which just takes that const char* and doesn't know anything about an underlying NSString or about freeing anything. Note: I have zero experience with objective-c)

Reply:
I have no idea obviously, but as that function is returning `const char*` and not NSString, how does the calling C++ code know that there's anything to free?
(I don't know how  the obj-c/obj-c++ garbage collector works, maybe it's handled automagically. Or maybe it's allocated in basically a static variable that belongs to OSX_GetLocalizedString() that is reused on next call so it doesn't leak endlessly, or soemthing else entirely)